src/GroupManager.php

```diff
@@ -93,6 +101,10 @@ public function addGroup($entity_type_id, $bundle_id) {
     $editable->set('groups', $groups);
     $saved = $editable->save();
 
+    // Notify other module we added a new group.
+    // todo: should this be an event?
+    $this->moduleHandler->invokeAll('og_group_created', [$entity_type_id, $bundle_id]);
```

Comment:
1. It is easy to call a class member from a hook.
1. Then you will test said class
1. Same
1. So it's not a problem as it has not been for 15 years
1. That is incredibly rare
1. There *is* compelling reason to do so, have you seen the amount of boilerplate an event listener requires?